app/AccountancyModule/presenters/CashbookExportPresenter.php

```diff
@@ -175,7 +177,7 @@ public function actionExportCashbookWithCategories(string $cashbookId, string $p
         $this->sendResponse(new ExcelResponse('pokladni-kniha', $spreadsheet));
     }
 
-    public function actionDownloadScan(string $cashbookId, int $chitId, string $path) : void
+    public function actionDownloadScan(string $cashbookId, int $chitId, string $path, bool $thumbnail = false) : void
```

Comment:
Tady by IMO stálo za zvážení, jestli nemít místo bool flagu, u kterýho je na volající straně těžko poznat, co dané `true`/`false` znamená, dvě různé akce - `actionDownloadScan()` a `actionDownloadScanThumbnail()` A velká část kódu se dá oddelegovat na privátní metodu.

Jinak bych si tady klidně zjendodušil práci a místo toho, aby ty akce byly v `CashbookExportPresenter` je přidal jako signal handlery přímo do `ChitScan`. To by to používání zjednodušilo na:

```php
public function handleDownload(string $path) : void;

public function handleDownloadThumbnail(string $path): void;
```

Reply:
Upravím to zatím jen na 2 actions, aby to bylo dostupné i mimo komponentu.

Reply:
Ok